core/templates/pages/learner-dashboard-page/goals-tab.component.ts

```diff
@@ -206,7 +208,7 @@ export class GoalsTabComponent implements OnInit {
       targetElement &&
       !this.dropdownRef.nativeElement.contains(targetElement)
     ) {
-      this.showThreeDotsDropdown = false;
+      this.showThreeDotsDropdown[0] = false;
```

Comment:
Won't you need to do this for all the buttons and not just the first one?

Reply:
Ya, Done for all now.